pkg/kubernetes/informers.go

```diff
@@ -101,6 +119,9 @@ func (im *InformerManager) AddConfigMapListener(add func(obj interface{}), updat
 		UpdateFunc: update,
 		DeleteFunc: remove,
 	})
+	stopCh := make(chan struct{})
+	//Since NewFilteredConfigMapInformer is not part of the informer factory, we need to invoke the Run() explicitly to start the shared informer
+	go im.configMapInformer.Run(stopCh)
```

Comment:
If this `Run` returns, then looks like the informer will stop watching on configmaps?

Reply:
> informer will stop watching on configmaps
The informer will be stopped when stopCh is closed.
Quoting the above from: https://github.com/kubernetes/client-go/blob/master/tools/cache/shared_informer.go#L158